### PR TITLE
Implement a reusable method for handle responses

### PR DESCRIPTION
**Pull request by SavidyaSathyani**

### CODE REVIEW

File: src/api/users.js [hunk 1]
```diff
@@ -41,14 +34,17 @@ export const all = (event, context, callback) => {
           },
         }).then((result) => {
           result = {data: result};
-          return done(null, result);
+          return callback(null, done(null, result));
         }).catch((error) => {
-          return done(error);
+          error.statusCode = 403;
```
Comment (singhabahu) [inline src/api/users.js:39]: In other cases like this it should be 500 internal server error right?

**Review by singhabahu — CHANGES_REQUESTED** [review]
@Rishanthakumar this looks good to me, I think we can merge this after fixing invalid response codes

File: src/api/users.js (hunk 1 above)
Comment (Rishanthakumar) [reply src/api/users.js:39]: @singhabahu  http://restcookbook.com/HTTP%20Methods/400-vs-500/ is this what you mentioning?

**Comment by Rishanthakumar** [comment]
@singhabahu yes this looks good.

**Comment by Rishanthakumar** [comment]
The requested changes will be done in the future commits.

**Pull request merged**